agentic-form-filler/app.py

```
@@ -0,0 +1,370 @@
+    st.subheader("📚 Recent Forms")
+    history = get_session_history()
+    if not history:
+        st.caption("No previous forms found.")
+    else:
+        for item in history[:8]: # Show last 8 sessions
+            t_str = ""
+            if item["time"]:
+                try:
+                    dt = datetime.datetime.fromisoformat(item["time"])
+                    t_str = dt.strftime("%b %d, %H:%M")
+                except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)